test/integration/master/synthetic_master_test.go

```diff
@@ -225,6 +225,72 @@ func TestStatus(t *testing.T) {
 	}
 }
 
+var expected1 = `etcdserver: request is too large`
+var expected2 = `rpc error: code = ResourceExhausted desc = grpc: trying to send message larger than max (3489913 vs. 2097152)`
+
+func TestObjectSizeResponses(t *testing.T) {
+
+	_, s, closeFn := framework.RunAMaster(nil)
+	defer closeFn()
+	transport := http.DefaultTransport
+
+	content1, err := ioutil.ReadFile("dummyfile1.txt")
+	if err != nil {
+		t.Fatalf("Unexpected error %v", err)
+	}
+
+	content2, err := ioutil.ReadFile("dummyfile2.txt")
```

Comment:
what exactly is in these files? (Github does not show them). Are that namespace objects?

Reply:
@sttts These are the request body that is of huge size, I was thinking of updating this way to generate this body, will update this